enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -5,21 +5,22 @@
 
 from django.db import models
 from django_extensions.db.models import TimeStampedModel
+from edx_enterprise_subsidy_client import EnterpriseSubsidyAPIClient
 from simple_history.models import HistoricalRecords
 
 from enterprise_access.apps.api.utils import acquire_subsidy_policy_lock, release_subsidy_policy_lock
 from enterprise_access.apps.api_client.discovery_client import DiscoveryApiClient
-from enterprise_access.apps.api_client.enterprise_catalog_client import EnterpriseCatalogApiClient
 from enterprise_access.apps.api_client.lms_client import LmsApiClient
 from enterprise_access.apps.subsidy_access_policy.constants import (
     CREDIT_POLICY_TYPE_PRIORITY,
-    SUBSCRIPTION_POLICY_TYPE_PRIORITY,
     AccessMethods
 )
-from enterprise_access.apps.subsidy_access_policy.mocks import group_client, subsidy_client
+
 
 SUBSIDY_POLICY_LOCK_TIMEOUT_SECONDS = 300
 
+SUBSIDY_CLIENT = EnterpriseSubsidyAPIClient()
```

Comment:
Actually, since you don't use this anywhere, but at the same time you removed the mock `subsidy_client` import, I'm guessing this is still a WIP PR.